core/src/main/java/com/sequenceiq/cloudbreak/controller/StackCreatorService.java

```diff
@@ -423,6 +422,8 @@ private Blueprint determineBlueprint(StackV4Request stackRequest, Workspace work
                     baseImageEnabled,
                     user,
                     image -> true);
+            MDCBuilder.cleanupMdc();
```

Comment:
please check if this cleanup is needed

Reply:
Yes we need this because we use a thread pool and other totally unrelated tasks will run on the same thread in the future.